src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from "./Checkbox.module.scss";
+
+type CheckboxProps = {
+  checked: boolean;
+  onChange: () => void;
+  className?: string;
+};
+
+const Checkbox = ({ checked, onChange, className }: CheckboxProps) => {
+  return (
+    <div className={styles["checkbox-wrapper"]}>
+      <label className={styles["checkbox-label-wrapper"]}>
+        <input
+          type="checkbox"
+          onChange={onChange}
+          checked={checked}
+          className={styles["task-item-checkbox"]}
```

Comment:
please use cammelCase for naming of CSS classes

Reply:
done